input/fsh/FHIRcastDiagnosticReportClose.fsh

```diff
@@ -11,7 +11,7 @@ In the FHIR R4 DiagnosticReport resource image study references would be placed
 
 In FHIRcast deployments based on FHIR R5, the attribute `study` SHALL be used rather than the `imagingStudy` attribute.
 
-Additionally FHIR R5 includes a `supportingInfo` attribute. While not yet formally provided for in FHIR R5, it has been recommended that the next release of FHIR allow an ImagingStudy reference be included in this attribute so that the DiagnosticReport could indicate one or more image studies were consulted during the creation of the report. As such in FHIR R5 deployments, this field is considered labeled as must support.
+Additionally FHIR R5 includes a `supportingInfo` attribute. While not yet formally provided for in FHIR R5 (R6 formalizes this support), it has been recommended that the next release of FHIR allow an ImagingStudy reference be included in this attribute so that the DiagnosticReport could indicate one or more image studies were consulted during the creation of the report. As such in FHIR R5 deployments, this field should be considered labeled as must support.
```

Comment:
R6's supportingInfo doesn't have an allowed reference of ImagingStudy  -- https://www.hl7.org/fhir/diagnosticreport.html

Reply:
My bad, I'm wrong